docs/source/nodes_and_pipelines/pipeline_introduction.md

```diff
@@ -34,12 +38,112 @@ variance_pipeline = pipeline(
     ]
 )
 ```
-</details>
 
```

Comment:
A little explanation here about how Kedro can figure out the order of execution of these nodes based on the inputs and outputs of the nodes in a pipeline.